source/appModules/soffice.py

```diff
@@ -433,6 +475,36 @@ def _backspaceScriptHelper(self, unit: str, gesture: inputCore.InputGesture):
 		else:
 			log.warning('Backspace did not remove text as expected.')
 
+	@script(
+		gestures=[
+			# bold
+			"kb:control+b",
+			# italic
+			"kb:control+i",
+			# underline
+			"kb:control+u",
+			# superscript
+			"kb:control+shift+p",
+			# subscript
+			"kb:control+shift+b",
+			# align left
+			"kb:control+l",
+			# align center
+			"kb:control+e",
+			# align right
+			"kb:control+r",
+			# justified
+			"kb:control+j",
+		]
+	)
+	def script_toggleTextAttribute(self, gesture: inputCore.InputGesture):
+		# reset time and enable announcement of toggled toolbar buttons
+		# see L{SymphonyButton.event_stateChange}

```

Comment:
Now that you are using `@script` decorator, you can safely use the docstring for this function instead of comments.

```suggestion
		"""Reset time and enable announcement of toggled toolbar buttons.
		See L{SymphonyButton.event_stateChange}
		"""
```

Reply:
Thanks!